crates/holochain/src/conductor/conductor.rs

```diff
@@ -1002,6 +1018,9 @@ mod builder {
         #[cfg(any(test, feature = "test_utils"))]
         /// Optional handle mock (for testing)
         pub mock_handle: Option<MockConductorHandleT>,
+        #[cfg(any(test, feature = "test_utils"))]
+        /// Optional test interface.
+        pub test_interface: Option<tokio::sync::broadcast::Sender<Signal>>,
```

Comment:
I'm unclear on the naming here. Can you shed some light on why this can be called a test interface?